include/picongpu/param/mallocMC.param

```diff
@@ -57,11 +59,11 @@ namespace picongpu
      * algorithm.
      */
     using DeviceHeap = mallocMC::Allocator<
-        cupla::Acc,
+        pmacc::Acc<DIM1>,
```

Comment:
Note, in general mallocMC needs the device and is taking the device type from the accelerator.
MallocMC needs a refactoring to cleanly reflect this, therefore currently we pass a one dimensional accelerator, which is fine.